### ISSUE TITLE
Xpand 4.222.0 and DX 22.2.4 crashes on startup

### ISSUE THREAD

**Issue by pjcunningham** [eXpandFramework/eXpand#982]
𝗗𝗲𝘀𝗰𝗿𝗶𝗯𝗲 𝘁𝗵𝗲 𝗯𝘂𝗴
Running a clean XAF Blazor project on Windows results in the following error:
```csharp
The error occurred:

	Type:       ArgumentNullException
	Message:    Value cannot be null. (Parameter 'context')
	Data:       0 entries
	Stack trace:

   at System.Reactive.Linq.Observable.ObserveOn[TSource](IObservable`1 source, SynchronizationContext context) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable.Concurrency.cs:line 62
   at Xpand.XAF.Modules.Reactive.Services.XafApplicationRxExtensions.WhenWindowCreated(XafApplication application, Boolean isMain, Boolean emitIfMainExists)
   at Xpand.XAF.Modules.Reactive.Services.XafApplicationRxExtensions.WhenSynchronizationContext(XafApplication application)
   at Xpand.XAF.Modules.Reactive.Services.ShowMessageExtensions.ShowMessages(XafApplication application)
   at Xpand.XAF.Modules.Reactive.RxApp.Connect(XafApplication application)
   at Xpand.XAF.Modules.Reactive.RxApp.<>c__DisplayClass12_0.<Connect>b__0(XafApplication application)
   at Xpand.XAF.Modules.Reactive.Services.ApplicationModulesManagerService.<>c__DisplayClass2_1`1.<WhenApplication>b__2()
   at System.Reactive.Linq.ObservableImpl.Defer`1._.Run() in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Defer.cs:line 37
	InnerException is null
```

𝗧𝗼 𝗥𝗲𝗽𝗿𝗼𝗱𝘂𝗰𝗲
1. Create an XAF project using the wizard in VS (XPO, Blazor)
2. Add Xpand nuget packages to project, (Xpand.XAF.Web.All, Xpand.XAF.Core.All, System.Interactive)
3. Configure `Startup.cs`:
```csharp
[assembly: HostingStartup(typeof(Xpand.Extensions.Blazor.HostingStartup))]
[assembly: HostingStartup(typeof(Xpand.XAF.Modules.Blazor.BlazorStartup))]
```
4. Register an Xpand module e.g. in `BlazorModule.cs`
```csharp
using System.ComponentModel;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;

namespace XafXpand2224.Blazor.Server;

[ToolboxItemFilter("Xaf.Platform.Blazor")]
// For more typical usage scenarios, be sure to check out https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.ModuleBase.
public sealed class XafXpand2224BlazorModule : ModuleBase {
    public XafXpand2224BlazorModule() {
        RequiredModuleTypes.Add(typeof(Xpand.XAF.Modules.Reactive.ReactiveModule));
        RequiredModuleTypes.Add(typeof(Xpand.XAF.Modules.RefreshView.RefreshViewModule));
    }
    public override IEnumerable<ModuleUpdater> GetModuleUpdaters(IObjectSpace objectSpace, Version versionFromDB) {
        return ModuleUpdater.EmptyModuleUpdaters;
    }
    public override void Setup(XafApplication application) {
        base.Setup(application);
    }
}
```
5. Run Web project, initially the navigation screen is shown and then crashes after a few seconds.

[XafXpand2224.zip](https://github.com/eXpandFramework/eXpand/files/10755798/XafXpand2224.zip)

𝗘𝘅𝗽𝗲𝗰𝘁𝗲𝗱 𝗯𝗲𝗵𝗮𝘃𝗶𝗼𝗿
The project to run

❤ Bronze Sponsor

**Comment by apobekiaris** [comment]
see #978

**Comment by apobekiaris** [comment]
there are still issues with ME, I am working on them

**Comment by apobekiaris** [comment]
The pre-release [4.222.2.0](https://github.com/eXpandFramework/Reactive.XAF/releases/tag/4.222.2.0) in the [Reactive.XAF](https://github.com/eXpandFramework/Reactive.XAF/tree/lab) `lab` branch  includes commits that relate to this task:

* [Fix wrong namespaces #978 #982](https://github.com/eXpandFramework/Reactive.XAF/commit/7dc1e463680c3e269013f27a1fa66ba72204e4b0)

To minimize version conflicts we recommend that you use the [Xpand.XAF.Core.All](https://www.nuget.org/packages/Xpand.XAF.Core.All), [Xpand.XAF.Win.All](https://www.nuget.org/packages/Xpand.XAF.Win.All), [Xpand.XAF.Web.All](https://www.nuget.org/packages/Xpand.XAF.Web.All) packages. Doing so, all packages will be at your disposal and .NET will add a dependecy only to those packages that you actually use and not to all (see the [Modules installation-registrations](https://youtu.be/LvxQ-U_0Sbg) youtube video).
<details>
    <summary>Released packages:</summary>
<a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions'>Xpand.Extensions v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Blazor'>Xpand.Extensions.Blazor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Mono.Cecil'>Xpand.Extensions.Mono.Cecil v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Office.Cloud'>Xpand.Extensions.Office.Cloud v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Office.Cloud.Google.Blazor'>Xpand.Extensions.Office.Cloud.Google.Blazor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Reactive'>Xpand.Extensions.Reactive v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.XAF'>Xpand.Extensions.XAF v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.XAF.Xpo'>Xpand.Extensions.XAF.Xpo v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib.Blazor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib.Common v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib.EasyTest v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/tools/Xpand.VersionConverter'>Xpand.VersionConverter v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules'>Xpand.XAF.Core.All v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/AutoCommit'>Xpand.XAF.Modules.AutoCommit v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Blazor'>Xpand.XAF.Modules.Blazor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/BulkObjectUpdate'>Xpand.XAF.Modules.BulkObjectUpdate v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/CloneMemberValue'>Xpand.XAF.Modules.CloneMemberValue v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/CloneModelView'>Xpand.XAF.Modules.CloneModelView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Email'>Xpand.XAF.Modules.Email v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/GridListEditor'>Xpand.XAF.Modules.GridListEditor v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/HideToolBar'>Xpand.XAF.Modules.HideToolBar v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/JobScheduler.Hangfire'>Xpand.XAF.Modules.JobScheduler.Hangfire v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/JobScheduler.Notification'>Xpand.XAF.Modules.JobScheduler.Notification v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/MasterDetail'>Xpand.XAF.Modules.MasterDetail v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ModelMapper'>Xpand.XAF.Modules.ModelMapper v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ModelViewInheritance'>Xpand.XAF.Modules.ModelViewInheritance v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.Cloud.Google'>Xpand.XAF.Modules.Office.Cloud.Google v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.Cloud.Google.Calendar'>Xpand.XAF.Modules.Office.Cloud.Google.Calendar v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.Cloud.Google.Tasks'>Xpand.XAF.Modules.Office.Cloud.Google.Tasks v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.DocumentStyleManager'>Xpand.XAF.Modules.Office.DocumentStyleManager v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/OneView'>Xpand.XAF.Modules.OneView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/PositionInListView'>Xpand.XAF.Modules.PositionInListView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ProgressBarViewItem'>Xpand.XAF.Modules.ProgressBarViewItem v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/RazorView'>Xpand.XAF.Modules.RazorView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive'>Xpand.XAF.Modules.Reactive v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Logger'>Xpand.XAF.Modules.Reactive.Logger v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Logger.Client.Win'>Xpand.XAF.Modules.Reactive.Logger.Client.Win v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Logger.Hub'>Xpand.XAF.Modules.Reactive.Logger.Hub v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Rest'>Xpand.XAF.Modules.Reactive.Rest v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/RefreshView'>Xpand.XAF.Modules.RefreshView v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/SequenceGenerator'>Xpand.XAF.Modules.SequenceGenerator v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Speech'>Xpand.XAF.Modules.Speech v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/SpellChecker'>Xpand.XAF.Modules.SpellChecker v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/StoreToDisk'>Xpand.XAF.Modules.StoreToDisk v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/SuppressConfirmation'>Xpand.XAF.Modules.SuppressConfirmation v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/TenantManager'>Xpand.XAF.Modules.TenantManager v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ViewEditMode'>Xpand.XAF.Modules.ViewEditMode v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ViewItemValue'>Xpand.XAF.Modules.ViewItemValue v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ViewWizard'>Xpand.XAF.Modules.ViewWizard v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Windows'>Xpand.XAF.Modules.Windows v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules'>Xpand.XAF.Web.All v.4.222.2</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules'>Xpand.XAF.Win.All v.4.222.2</a></br>
</details>

Please update the related Nuget packages and test if issues is addressed. These are nightly nuget packages available only from our [NugetServer](https://xpandnugetserver.azurewebsites.net/nuget/).

If you do not use these packages directly but through a module of the main eXpandFramework project, please wait for the bot to notify you again when integration is finished or update the related packages manually.

Thanks a lot for your contribution.

**Comment by pjcunningham** [comment]
That's fixed it!
